# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/WindowGlobalParent.h
# Commit: ccd96b99aea2
# Full Hash: ccd96b99aea28e8214e585b864d3fb2245bac142
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-06-07 08:31:23
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing failures complaining about fedcm-no-registered-idps.https.html.
#   
#   This reverts commit ba33457e0cbac24a8b7d963c45138c62e92a74ae.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 4bfadfc33207 -r ccd96b99aea2 dom/ipc/WindowGlobalParent.h
--- a/dom/ipc/WindowGlobalParent.h	Fri Jun 06 21:07:41 2025 +0300
+++ b/dom/ipc/WindowGlobalParent.h	Fri Jun 06 21:35:18 2025 +0300
@@ -318,6 +318,22 @@
 
   mozilla::ipc::IPCResult RecvReloadWithHttpsOnlyException();
 
+  mozilla::ipc::IPCResult RecvGetIdentityCredential(
+      IdentityCredentialRequestOptions&& aOptions,
+      const CredentialMediationRequirement& aMediationRequirement,
+      bool aHasUserActivation, const GetIdentityCredentialResolver& aResolver);
+  mozilla::ipc::IPCResult RecvStoreIdentityCredential(
+      const IPCIdentityCredential& aCredential,
+      const StoreIdentityCredentialResolver& aResolver);
+  mozilla::ipc::IPCResult RecvDisconnectIdentityCredential(
+      const IdentityCredentialDisconnectOptions& aOptions,
+      const DisconnectIdentityCredentialResolver& aResolver);
+  mozilla::ipc::IPCResult RecvSetLoginStatus(
+      LoginStatus aStatus, const SetLoginStatusResolver& aResolver);
+
+  mozilla::ipc::IPCResult RecvPreventSilentAccess(
+      const PreventSilentAccessResolver& aResolver);
+
   mozilla::ipc::IPCResult RecvGetStorageAccessPermission(
       bool aIncludeIdentityCredential,
       GetStorageAccessPermissionResolver&& aResolve);
@@ -334,8 +350,6 @@
   already_AddRefed<dom::PWebAuthnTransactionParent>
   AllocPWebAuthnTransactionParent();
 
-  already_AddRefed<dom::PWebIdentityParent> AllocPWebIdentityParent();
-
  private:
   WindowGlobalParent(CanonicalBrowsingContext* aBrowsingContext,
                      uint64_t aInnerWindowId, uint64_t aOuterWindowId,